cas/incident-report.md

```diff
@@ -1,169 +1,387 @@
+
+**Note:** The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" on this page are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+### Why is public Incident Reporting considered important?
+
+The public Incident Reporting process is important because it promotes continuous improvement, information sharing, and highlights opportunities to define and adopt improved practices, policies, and controls. Together, these activities help build a more secure web.
+
+### Who can submit an Incident Report?
+
+Anyone should feel encouraged to submit an Incident Report that’s founded upon credible and well-substantiated evidence.
+
+Some Root Store Operator policies require CA Owners to submit an Incident Report as described on this page after self-discovering or being made aware of an Incident (e.g., receiving and corroborating an issue described in a [Certificate Problem Report](https://cabforum.org/working-groups/server/baseline-requirements/requirements/#161-definitions)).
+
+### Are there other ways to become involved in the Incident Reporting process?
+
+Absolutely! There are many ways to participate in the Incident Reporting process beyond submitting new reports. Everyone is encouraged to actively contribute by commenting on existing reports and engaging in constructive discussions. This can include, but is not limited to:
+
+- Providing additional information,
+- Asking clarifying questions,
+- Discussing technical aspects of the incident,
+- Suggesting corrective actions,
+- Highlighting opportunities for improvement,
+- Contributing lints to open source linting projects to promote improved future issue detection/prevention, and
+- Sharing lessons learned from past experiences.
+
+Individuals representing CA Owners are especially encouraged to participate broadly in the Incident Reporting process, extending their contributions beyond incidents involving only their own organization. Sharing insights and perspectives across organizational boundaries fosters a collaborative learning environment and strengthens the overall security posture of the Web PKI ecosystem.
+
+To ensure productive dialogue, please keep all comments constructive, relevant to the report, and in line with the [CCADB Code of Conduct](https://docs.google.com/document/d/19ALqEvHtTE6OUTz2FaOXrU9gruIdvia5EDh3hXeGpZA/edit#heading=h.cumc0pgd1s7c).
+
+### How do I submit an Incident Report?
+Create a new Bugzilla issue by filling out the Summary and Description fields of [this form](https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&product=CA%20Program&component=CA%20Certificate%20Compliance&bug_type=task). 
+
+The "Summary" field in Bugzilla (i.e., "Subject line") MUST begin with the CA Owner’s name, followed by a colon, and a brief title that highlights the type of incident being reported (e.g., "EXAMPLE CA OWNER: Incorrect Subject RDN Encoding"). 
```

Comment:
What is the expectation for the "CA Owner" value? Does it need to match exactly with the CA Owner (casing, use of diacritics, non-ASCII characters etc.) value in CCADB?

Reply:
Clarified in #187. 

> The CA Owner's name SHOULD match exactly with the CA Owner value in the CCADB.